akka-typed/src/main/scala/akka/typed/ActorRef.scala

```diff
@@ -9,6 +9,13 @@ import language.implicitConversions
 import scala.concurrent.Future
 
 /**
+ * Helper parent to allow [[ActorRef]] to be used as key in TypedMultiMap.
```

Comment:
Ah, reminds me of the discussion with Martin some time ago. Looking forward to scroll down and find that map then...